Fix #10: feat(frontend): add profile-card component

harishpullagurla opened the pull request:
## Overview
<!--
READ ME FIRST:
Please answer *both* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes #10 .
2. This PR does the following: Implements a reusable ProfileCard component to display user profiles. This component will be used on the home page to render profile data.
   

## Essential Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [x] The PR is made from a branch that's **not** called "main/master".

## Proof that changes are correct

<!--
Add videos/screenshots of the user-facing interface to demonstrate that the changes
made in this PR work correctly. If this PR is for a developer-facing feature,
provide the videos/screenshots of developer-facing interface.

Please also include videos/screenshots of the developer tools
browser console, so that we can be sure that there are no console errors.

If the changes in your PRs are autogenerated via a script and you cannot provide proof
for the changes then please leave a comment "No proof of changes needed because {{Reason}}"
and remove all the sections below.
-->
![WhatsApp Image 2025-06-11 at 16 15 11_f22c303d](https://github.com/user-attachments/assets/bdfcb9e5-aa3a-495d-bf49-a9def8d1963c)
![WhatsApp Image 2025-06-11 at 16 15 59_cd731414](https://github.com/user-attachments/assets/970190eb-31ec-4a10-921f-8ff6ca02cc0f)

Zapper9982 commented:
can u link the PR to the issue , please , 

do it by changing the first part in the overview  from :

This PR fixes or fixes part of #[10]. to ------> This PR fixes #10.

By this Github understands to link the pull request to the issue ... Also do keep this in mind for the next time :)

Zapper9982 commented:
@Om-Thorat  can u check if the font is right

Om-Thorat reviewed (changes requested):
Good work @harishpullagurla :) , Just little detailes might be missing, can you loook at the Figma design and adjust accordingly. Using Figma is a valuable skill as well its the primary design tool in virtually all corporations. here is the link https://www.figma.com/design/RjhX2ZPg1779ONIRc72nEN/YDO-Github?node-id=89-2

harishpullagurla commented:
Updated the ProfileCard component based on Figma design .
Adjusted layout, spacing, and styling as per feedback.
Please review and let me know if any further changes are required.
![WhatsApp Image 2025-06-14 at 15 08 17_233b334c](https://github.com/user-attachments/assets/562f6277-d357-4d9c-8369-662249d3fa4f)
this is image of the card

Om-Thorat commented on package-lock.json:
```diff
@@ -5419,6 +5419,21 @@
         "url": "https://github.com/sponsors/sindresorhus"
       }
     },
+    "node_modules/typescript": {
```
Is this intentional? please remove if irrelevant.

Om-Thorat commented:
Congratulations! and thank you for contributing :)
